### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: adrian/script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  favoriteFoods: ["tomatoes", "cucumber", "flies"],
+
+  //2.4
+  printFood: function () {
+    this.favoriteFoods.map((food) => console.log(food));
+  },
+};
+//2.3
+let secondElement = doggo.favoriteFoods[1];
+
```
Comment (ciamac-da) [inline adrian/script.js:27]: Please call the function to print all the doggo's favorite food. 
  doggo.printFood()

File: adrian/script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  printIngredient() {
+    for (const key in this.ingredients) {
+      console.log(this.ingredients[key]);
+    }
+  },
+};
+//3.3
+recipes.ingredients.ginger = "ginger";
+//3.4
+recipes.ingredients.sugar = "brown sugar";
```
Comment (ciamac-da) [inline adrian/script.js:47]: Please call the function to print  the value of each ingredient in the ingredients object

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: adrian/script.js (hunk 1 above)
Comment (AdrianValdes) [reply adrian/script.js:27]: I did it, but the exercise doesn't say I have to call it, it says: "2.4 Add a method that loops through and print all the doggo's favorite food." That is why I didn't call it.

File: adrian/script.js (hunk 2 above)
Comment (AdrianValdes) [reply adrian/script.js:47]: It did it, but the exercise doesn't say I have to call it, it says: "3.5 Write a method in the recipes object that prints the value of each ingredient in the ingredients object." That is why I didn't call it.

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**